src/vm/eeconfig.cpp

```diff
@@ -789,10 +792,18 @@ HRESULT EEConfig::sync()
         g_fEnableARM = TRUE;
     }
 
-    int forceGCconcurrent = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_gcConcurrent);
-    if ((forceGCconcurrent > 0) || (forceGCconcurrent == -1 && g_IGCconcurrent))
+    bool gcConcurrentWasForced = false;
+#ifdef FEATURE_CORECLR
```

Comment:
I do not think we need this change and the related change in clrconfig.h.

What will not  work if both these changes are reverted?